api/src/main/java/com/velocitypowered/api/proxy/config/ProxyConfig.java

```diff
@@ -62,6 +62,15 @@
    */
   boolean isOnlineMode();
 
+  /**
+   * If client's ISP/AS sent from this proxy is different from the one from Mojang's"
```

Comment:
Errant `"`

Reply:
Yeah copied from the configuration class, oops.